hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AAnalyticsAcceleratorStreamReading.java

```diff
@@ -23,8 +23,8 @@
 import java.io.IOException;
 import java.io.InputStream;
 
```

Comment:
what other test cases can you think of that can make these GET assertions useful?

For example, if we read an object of size ~10MB, can we assert that we made only 2 GETs? 
Similarly if read a small object, can we assert there was only a single GET?
If we passed in the object lenght when opening the object, can we assert that there were 0 HEADs?
If there are concurrent streams reading the same object, can we assert that there are no duplicate GETs?